sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -551,3 +551,13 @@ def test_load_table_fits(self):
         data = ui.get_data(1)
         self.assertEqualWithinTol(data.x, [1, 2, 3])
         self.assertEqualWithinTol(data.y, [4, 5, 6])
+
+
+@requires_fits
+def test_bug_276(make_data_path):
+    ui.load_pha(make_data_path('3c273.pi'))
+    ui.set_model('polynom1d.p1')
+    ui.fit()
+    ui.covar()
+    scal = ui.get_covar_results().parmaxes
+    ui.sample_flux(ui.get_model_component('p1'), 0.5, 1, num=5, correlated=False, scales=scal)
```

Comment:
Since we are talking about review practices, one of the main advantages of working on the same repo rather than on individual forks is that people can push commit to a PR, especially in simple cases like this.

I would encourage us all to do it. It's good to coordinate this kind of changes, i.e. leaving a review comment saying: "there should be a newline at the end of file, I am going to push a commit to fix that".

Of course, it's just a suggestion, and I don't think it makes any sense to turn this into a policy. Just a reminder we can take advantage of the decision to work on the same repo.